src/Tools/ReleaseNotesUtil/ReleaseNotesUtil.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
Consider updating this line:

https://github.com/dotnet/roslyn-analyzers/blob/dc672ef14b6c6ce5db6e9c0ee39e47e07c9f8510/PostReleaseActivities.md#L28